src/Utilities/CssConcatenator.php

```diff
@@ -98,19 +91,16 @@ public function getCss(): string
      * @param string $media The media query for rules to be appended, e.g. "@media screen and (max-width:639px)",
      *                      or an empty string if none.
      *
-     * @return \stdClass Object with properties as described for elements of `$mediaRules`.
+     * @return CssConcatenator\MediaRule Object with properties as described for elements of `$mediaRules`.
      */
-    private function getOrCreateMediaRuleToAppendTo(string $media): \stdClass
+    private function getOrCreateMediaRuleToAppendTo(string $media): CssConcatenator\MediaRule
     {
         $lastMediaRule = \end($this->mediaRules);
         if ($lastMediaRule !== false && $media === $lastMediaRule->media) {
             return $lastMediaRule;
         }
 
-        $newMediaRule = (object)[
-            'media' => $media,
-            'ruleBlocks' => [],
-        ];
+        $newMediaRule = new CssConcatenator\MediaRule($media);
```

Comment:
constructor defaults ruleBlocks to `[]`, so passing both in seemed redundant, as the change couldn't be described with a 2-line diff without having the constructor first accept an array.

Reply:
Absolutely.

Something's broken with Firefox + Github.  I already commented here.

We only had the `[]` initialization because we were using an object-like-array, AFAICR.

Reply:
possibly not firefox, I had an issue with a review comment not posting in chrome.

Reply:
Both my comments are now here, but one of them is somewhere else.  Weird?  Maybe because this conversation has been copied to another place, or because a certain commit has been rebased and the review comments have been copied to another PR.  This is confusing the H out of me.

I've literally just seen the other one, but closed the tab, and even with "Undo Closed Tabs Button" addon cannot revive it.

I'd really like to be able to work without significant UX bugs in the tools used, i.e. GitHub.